src/test/scala/mesosphere/marathon/integration/facades/MesosFacade.scala

```diff
@@ -61,7 +61,7 @@ object MesosFacade {
     def apply(vals: (String, Any)*): ITResources = {
       val resources: Map[String, ITResourceValue] = vals.map {
         case (id, value: Double) => id -> ITResourceScalarValue(value)
-        case (id, portsString: String) => id -> ITResourcePortValue(portsString)
+        case (id, portsString: String) => id -> ITResourceStringValue(portsString)
```

Comment:
Nit: should this be `value` instead of `postString` then?

Reply:
It should!